packages/ringcentral-widgets-cli/templates/Project/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.1",
+  "description": "A RingCentral Widgets App",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "start": "npm run dev-server",
+    "dev-server": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --inline",
+    "build": "NODE_ENV=production webpack --config webpack-production.config.js --progress --colors"
+  },
+  "devDependencies": {
+    "babel-eslint": "^7.2.3",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^14.1.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
+    "eslint-plugin-react": "^6.10.3",
+    "dotenv": "^4.0.0"
+  },
+  "dependencies": {
+    "autoprefixer": "^6.7.7",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.0.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "babel-runtime": "^6.23.0",
+    "classnames": "^2.2.5",
+    "copy-webpack-plugin": "^4.0.1",
+    "core-js": "^2.4.1",
+    "css-loader": "^0.28.1",
+    "file-loader": "^0.11.1",
+    "format-message": "^5.1.2",
+    "intl": "^1.2.5",
```

Comment:
intl should no longer be necessary. We should also remove this from our dependencies.